ui/components/institutional/compliance-modal/compliance-modal.js

```diff
@@ -35,11 +41,11 @@ const ComplianceModal = () => {
       <Box>
         <Box
           as="header"
-          display={DISPLAY.FLEX}
+          display={Display.Flex}
           justifyContent={JustifyContent.spaceBetween}
         >
           <Box
-            display={DISPLAY.FLEX}
+            display={Display.Flex}
             justifyContent={JustifyContent.flexStart}
             alignItems={AlignItems.center}
           >
```

Comment:
No visual regressions. Missing image is a build type issue that's not in production

<img width="1439" alt="Screenshot 2023-07-17 at 5 38 13 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/a97feb9b-d580-4c5a-a6f0-a95a5c5aa8d6">
<img width="1435" alt="Screenshot 2023-07-17 at 5 39 07 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/05bf8f6f-e660-4d5e-9712-df3a56d79169">